src/visualizers/VisualizerFactorHistogram.ts

```diff
@@ -0,0 +1,200 @@
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import {VisualizerDefault} from './VisualizerDefault'
+import {natlog} from '../shared/math'
+
+/** md
+# Factor Histogram
+
+(example image should go here)
+
+This visualizer takes the prime factorization of each element in the sequence
+and then creates a histogram of the number of elements of the sequence
+with X prime factors. the number of elements with X prime factors is along
+the x-axis. And the height is how many elements are in each bin of X.
+
+## Parameters
+**/
+
```

Comment:
As far as I can see, this is the only documentation comment. So the user guide entry will end with a "Parameters" header, and nothing listed under that. Recommend that you add more documentation comments next to each parameter saying what the parameter is/does.  And if you wanted to make a screenshot of your visualizer and check that in, too, and change the "(example image should go here)" to an actual link to the screenshot, that would be awesome (but it is definitely going the extra mile, don't feel obliged).

Reply:
I have everything else in this PR round done. And I was working on Inserting the Image. I currently added a screenshot in in the path 
\frontscope\src\assets\img\FactorHistogram\ExampleImage.png
How exatly do I link this in the documentation comment

Reply:
Hmm, should be
`![Factor Histogram Example](../assets/img/FactorHistogram/ExampleImage.png)`. I will pull and test, and just go ahead and push the fix if it works.

Reply:
Actually turns out to be a bit more complicated than that if you want the picture to be a reasonable size; so hang tight til I push a commit today or tomorrow.

Reply:
OK, that should do it.